### PR TITLE
feat(cdk): decouple `zone-configurations` into its own library

### PR DESCRIPTION
**Pull request by arturovt**
Michael, since we discussed that your PR is very huge, I'm splitting it into smaller PRs that are easier to review and maintain.

### CODE REVIEW

**Comment by BioPhoton** [comment]
Definitely! It was ment to get an idea of the sub packages

**Comment by BioPhoton** [comment]
Before we merge, check out if there are any changes in the big PR. I remember I did some cleaning. Especially to event namens.

**Comment by arturovt** [comment]
> Before we merge, check out if there are any changes in the big PR. I remember I did some cleaning. Especially to event namens.

Updated.

File: libs/cdk/zone-configurations/src/lib/zone-config.ts [hunk 1]
```diff
@@ -148,11 +129,7 @@ function createZoneFlagsConfigurator(): ZoneConfig {
 }
 
 /**
- * An object for typed zone-flags configuration. It exposes flags under the following scopes:
- * - global (global environment API)
- * - events (event listener names and behavior)
- * - runtime (configurable behaviour at runtime)
- * - test (patches for testing libs like jasmine, jest etc)
+ * An object for typed zone-flags configuration.
```
Comment (BioPhoton) [inline libs/cdk/zone-configurations/src/lib/zone-config.ts:132]: Why did you remove the comments here?

Comment (arturovt) [reply libs/cdk/zone-configurations/src/lib/zone-config.ts:132]: I didn’t, I just copied the file from your PR.

Comment (BioPhoton) [reply libs/cdk/zone-configurations/src/lib/zone-config.ts:132]: managed to update, just need to push to your PR later on

**Comment by arturovt** [comment]
Closing in favor of #728
